common/compositor/compositordefs.h

```diff
@@ -42,7 +45,15 @@ static float TransformMatrices[] = {
 };
 // clang-format on
 
-#ifdef USE_GL
+#ifdef USE_DC
+typedef unsigned GpuResourceHandle;
+typedef struct gl_import {
+  EGLImageKHR image_ = 0;
```

Comment:
do u need image_ ?

Reply:
I believe this is referenced outside of the USE_GL block in GetGpuResource(), I've been seeing similar issues with the Vulkan build.

Reply:
we should fix that

Reply:
You should have here void* image_ = 0; instead of EGLImageKHR